lib/src/main/res/layout/com_auth0_lock_changepwd_form_view.xml

```diff
@@ -46,11 +46,11 @@
         android:textSize="@dimen/com_auth0_lock_title_text" />
 
     <com.auth0.android.lock.views.ValidatedUsernameInputView
-        android:id="@+id/com_auth0_lock_input_username_email"
+        android:id="@+id/com_auth0_lock_input_email"
         android:layout_width="match_parent"
         android:layout_height="wrap_content"
         android:layout_marginLeft="@dimen/com_auth0_lock_widget_horizontal_margin"
         android:layout_marginRight="@dimen/com_auth0_lock_widget_horizontal_margin"
-        app:Auth0.InputDataType="username_or_email" />
+        app:Auth0.InputDataType="email" />
```

Comment:
instead of `app` let's use `lock` (or `auth0`) and `input_data_type` as the name